src/EA.Weee.Web/ViewModels/NewUser/UserCreationViewModel.cs

```diff
@@ -40,5 +40,8 @@ public class UserCreationViewModel
 
         [MustBeTrue(ErrorMessage = "Confirm that you've read and accepted the terms and conditions")]
         public bool TermsAndConditions { get; set; }
+
+        [MustBeTrue(ErrorMessage = "Confirm that you've read and accepted the privacy policy")]
+        public bool PrivacyPolicy { get; set; }
```

Comment:
Can we add unit test to the model for this attribute - and maybe the T & C one also